### DISCUSSION TITLE
Content menu items not rendering correctly in The Agency Theme.

### DISCUSSION THREAD

**Discussion by CADMANGO** [OrchardCMS/OrchardCore#9757] [General]
As there is no template for Content Menu Items in The Agency Theme it does not add the bootstrap attributes, so these items do not render in the same way as Link Menu Items.

I added the following template which resolves the display issue but it does not retrieve any url [empty value]:
[file: MenuItemLink-ContentMenuItem.liquid]
{% assign link = Model.ContentItem.Content.ContentMenuItemPart %}
<_a class="nav-link js-scroll-trigger" href="{{ link.Url | href }}">{{ link.Name }}</_a>
(Ignore the _ before the a tag. Only way I know to avoid it rendering as HTML in this post)

Can someone point me to the correct way to retrieve the url from a ContentMenuItem?
Any help will be greatly appreciated.

**Comment by ns8482e** [comment]
`ContentMenuItemPart` doesn't have `Url` I guess you mean to use   `LinkMenuItemPart`?
```liquid
{% assign link = Model.ContentItem.Content.LinkMenuItemPart %}
```
